docs/connection/connection-spans.md

```diff
@@ -0,0 +1,180 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: Connection Spans
+--->
+
+# Semantic Conventions for Connection Spans
+
+This document defines semantic conventions to apply when instrumenting client side of socket connections with spans.
```

Comment:
With http/3 over quick, the connection is virtual and may span multiple UDP packets. The client IP/Network may even change during the duration of the connection, for example switching between wifi and cellular when a mobile client is moved out of range.
Rather than tying this directly to a socket, the type can be tracked by an additional `type` property. This same concept can then be used for database, http and a range of scenarios, but with optional attributes based on the scenario.

Reply:
http/3 still operates on top of UDP sockets. 
I'm not an expert, but I believe from socket perspective we still have different connections established when QUIC connection migration happens, the only thing it saves is TLS handshake - it won't happen again during migration.

It's a good question how to represent QUIC logical connection, but given it's such a long loved thing, I don't see why we can't span for it and all the underlying socket connections it creates.